### PR TITLE
Adds Jupyter notebook with detailed instructions on creating GIFs from VEDA data holdings

### PR DESCRIPTION
**Pull request by leothomas**
Includes info on how to overlay a geojson raster, how to display the GIF over a basemap, and how to overlay the date

### CODE REVIEW

**Review by abarciauskas-bgse — APPROVED** [review]
This is awesome @leothomas - the content and details are awesome and pretty clear. My two suggestions for improvement:
1. extract some functions to separate python files, to make the notebook shorter and just focused on calling the functions themselves. 
2. Reduce the number of places we specify file names and paths. Sometimes filenames and their paths are duplicated and could be DRY'ed up so that if something changes in the directory structure or filename we only have to change it in one place instead of multiple places.

**Comment by leothomas** [comment]
@abarciauskas-bgse My apologies for the delay! I've removed some of the supporting code to a separate file with some more error handling, etc. and I've tried to reduce the number of places we specify filenames. I hope this addresses both of your feedback items. 

If you have some time, would you mind taking another look?

File: requirements.txt
Comment (abarciauskas-bgse) [inline requirements.txt]: I also had to 
```
pip3 install webdriver_manager
```
```suggestion
webdriver_manager
```

**Comment by abarciauskas-bgse** [comment]
Looks amazing @leothomas thank you for following up on this. 2 very minor things:

1. `This means that internal borders will be twice as thick as external borders` and `we want to add the date of each file` statements are missing a `.` 
2. In "Putting it all together" you switch from addressing the reader with the steps from "we" to "I". You of course did write the entire tutorial (which is so awesome 👏🏽 ) but probably is best to be consistent, and I imagine consistent with the other notebooks, to use "we"

>I've combined all of the above functionality...

could be: 

> Now we can combine all of the above functionality, along with a few helper functions in the file: `./gif_generation_dependencies/helper_functions.py` and add the contextualization steps (overlaying geojson, date, and folium basemap) directly into the generate_frame() method.

File: requirements.txt
Comment (abarciauskas-bgse) [reply requirements.txt]: I got a missing dependency error trying to load the helper files when it tries to run `from selenium import webdriver`

**Comment by abarciauskas-bgse** [comment]
I still approve this PR (no way to reapprove that I am aware of 😁 ), however if you agree we may be missing the `webdriver_manager` dependency that would be good to add. Not sure if it's just that my version of selenium was missing it. Looks like I have selenium `4.4.3`

**Comment by leothomas** [comment]
@abarciauskas-bgse I've added webdriver-manager as a dependency. This notebook should be good to go - once we merge it the CI will regenerate the HTML for the notebook

**Review by abarciauskas-bgse — APPROVED** [review]

**Pull request merged**
